tests/src/test/java/alluxio/hadoop/HdfsFileInputStreamIntegrationTest.java

```diff
@@ -53,25 +59,32 @@
 
   @BeforeClass
   public static final void beforeClass() throws Exception {
-    FileSystem fileSystem = sLocalAlluxioClusterResource.get().getClient();
-    FileSystemTestUtils
-        .createByteFile(fileSystem, "/testFile1", WriteType.CACHE_THROUGH, FILE_LEN);
-    FileSystemTestUtils.createByteFile(fileSystem, "/testFile2", WriteType.THROUGH, FILE_LEN);
+    sFileSystem = sLocalAlluxioClusterResource.get().getClient();
   }
 
   @After
-  public final void after() throws Exception {
+  public final void after() throws IOException, AlluxioException {
```

Comment:
I don't think `ClientTestUtils.resetClient()` resets any configuration. All configuration will eventually be reset at the end of each test by `LocalAlluxioClusterResource`. It isn't great how we modify the global conf during a test method and rely on it being reset by the cluster resource, but I don't think there's a much better way.